diagnostic.md

```diff
@@ -3,67 +3,59 @@
 Using `inspect element`, write four selectors found on the main page for this
 GitHub repo.
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+#ajax-error-message, .container, .js-stale-session-flash, #facebox
 
 ## Question 2
 
 Using the diagram in the `README` file, what are two ways you could return ALL
 list items (`li`) using jQuery? This is intentionally vague. (Assume starting
 at the `OL` DOM node).
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+jquery("li") and $("li")
 
 ## Question 3
 
 Write the jQuery code to navigate from one of the list item (`li`) DOM nodes to
 the `form` DOM node. (Assume the list item has a class of "listerItem").
 
-<!-- your answer starts here -->
 
-<!-- your answer ends here -->
 
 ## Question 4
 
 Give two examples of jQuery getter methods and two examples of jQuery setter
 methods.
 
-<!-- your answer starts here -->
+Getter
+$("h1").val()
+$("title").attr()
 
-<!-- your answer ends here -->
+Setter:
+$("h1").val("Help!")
+$("title").attr("Help!")
 
 ## Question 5
 
 In your own words, please answer the following question: WHAT IS THE DOM?
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+The DOM is a representation of the HTML using JavaScript objects. The DOM is created and loaded into the browser when the HTML is loaded and is subsequently made into Objects that can be manipulated using JavaScript. Each element is made into a Node which represents an object that can be manipulated using javascript methods. The manipulation allows for a more dynamic user experience.
 
 ## Question 6
 
 Please give two examples of things that are commonly confused for the DOM.
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+Two things that are typically confused for the DOM are the source HTML code and what is displayed in the elements tab in the Chrome inspector (it looks like the DOM and is derived from it but it is not the DOM).
 
 ## Question 7
 
 Please write a method that traverses from a list item (`li`) DOM node to the
 body DOM node, then hides it.
 
-<!-- your answer starts here -->
 
-<!-- your answer ends here -->
 
 ## Question 8
 
 Use jQuery to register a click handler for clicks inside `div` items.
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+$(‘div’).on(‘click’,(event => {
+        // does something
+});
```

Comment:
There needs to be a matching closing parenthesis here since you wrote the callback function with an opening parenthesis.